whisper_fastapi_online_server.py

```diff
@@ -201,24 +221,38 @@ async def start_ffmpeg_decoder():
     )
     return process
 
-async def transcription_processor(shared_state, pcm_queue, online):
+async def transcription_processor(shared_state, pcm_queue, transcriber):
+
+    
     full_transcription = ""
-    sep = online.asr.sep
```

Comment:
This didn't work when online was a VACOnlineProcessor.
But apparetnly no errow was raised